src/plasmapy/simulation/particle_integrators.py

```diff
@@ -235,13 +235,16 @@ def push(x, v, B, E, q, m, dt):
         .. [1] C. K. Birdsall, A. B. Langdon, "Plasma Physics via Computer
                Simulation", 2004, p. 58-63
         """
-
-        γ = 1 / np.sqrt(1 - (np.linalg.norm(v) / _c.si.value) ** 2)
+        γ = 1 / np.sqrt(
+            1 - (np.linalg.norm(v, axis=1, keepdims=True) / _c.si.value) ** 2
+        )
         uvel = v * γ
 
         uvel_minus = uvel + q * E * dt / (2 * m)
 
-        γ1 = np.sqrt(1 + (np.linalg.norm(uvel_minus) / _c.si.value) ** 2)
+        γ1 = np.sqrt(
+            1 + (np.linalg.norm(uvel_minus, axis=1, keepdims=True) / _c.si.value) ** 2
+        )
 
         # Birdsall has a factor of c incorrect in the definition of t?
```

Comment:
I forget, did we determine Birdsall actually had this factor of c wrong? Should this comment still be here?

Reply:
Good catch! iirc Birdsall was using CGS units